### ISSUE TITLE
Adding support for CN server if possible

### ISSUE THREAD

**Issue by Elypha** [Ottermandias/Glamourer#23]
Hello, 

Chinese server has just recently received endwalker update. I presume that differences in clients may have been much reduced as I noticed many plugins are working as expected on CN server without further modification. I'm wondering if you would like to maybe adapt it and make Glamourer also work on CN clients?

Currently there's an error upon installation which I think might be library path issues from what I can tell from a [fork](https://github.com/4679/Glamourer) which is partially working. `Fixed Design` in this fork is broken while others seem good so far.

**Comment by Ottermandias** [comment]
No, I won't. How would I? I do not have access to a chinese client, I do not have access to the data tables. I can not read it. I can not test anything.

**Comment by Elypha** [comment]
> No, I won't. How would I? I do not have access to a chinese client, I do not have access to the data tables. I can not read it. I can not test anything.

Thanks for reply. Yeah as you said I understand and I know it requires you to devote extra time. I was asking because I'm not sure if it would be possible to make it if I provide you with logs and help you to test things, or if it actually requires massive work that goes totally beyond such a workflow.
